projects/plugins/boost/app/modules/optimizations/minify/Concatenate_JS.php

```diff
@@ -97,7 +97,7 @@ public function do_items( $handles = false, $group = false ) {
 
 			// Only try to concat static js files
 			if ( false !== strpos( $js_url_parsed['path'], '.js' ) ) {
-				$do_concat = jetpack_boost_page_optimize_should_concat_js();
+				$do_concat = true;
```

Comment:
Just making sure, this isn't a debugging statement and $do_concat is always true under this condition?

Reply:
Yep, can confirm - it's not for debugging. I'll try to explain the reason behind it.

Previously the logic in the JS class could be used by enabling two separate options in the UI. One for concatenation and the other for the way JS is loaded (async, defer or default). With the removal of the second ("JS load mode"), this [function](https://github.com/Automattic/jetpack/pull/30005/files/a940835b6d1b91b58cab8781c907a18dc1785107#diff-924c0132da480bb9231bf48da1991027597c8ce38000fb7e4849944fb70d0258L74) was redundant as there was no need to check if the functionality is enabled within the functionality itself (since Boost does that outside the JS class now).